bookmarks/vanilla/src/modules/Bookmark.ts

```diff
@@ -217,10 +222,12 @@ export class Bookmark {
         );
     }
     private static mapLink(thing: ThingPersisted): string {
-        return (
-            getLiteral(thing, BOOKMARK.recalls)?.value ??
-            getNamedNode(thing, BOOKMARK.recalls)?.value ?? ""
-        );
+        const link = getLiteral(thing, BOOKMARK.recalls)?.value ?? getNamedNode(thing, BOOKMARK.recalls)?.value
+        if (link) {
```

Comment:
Can we check if it is also a valid link?

Reply:
yes sure, but let it be done in https://github.com/solid-contrib/data-modules/issues/32

Reply:
it needs more changes than a simple validation
we also need to update API schema in this case